scripts/setup_webserver.sh

```diff
@@ -166,9 +166,14 @@ EOF
     wait_for_apt_lock
     apt-get -y -qq -o=Dpkg::Use-Pty=0 install glusterfs-client
   elif [ "$fileServerType" = "azurefiles" ]; then
-    #apt-get -y install cifs-utils
-    wait_for_apt_lock
-    apt-get -y -qq -o=Dpkg::Use-Pty=0 install cifs-utils
+    if [ "$storageAccountType" = "Premium_LRS" ]; then
```

Comment:
does = work? == is not required?

Reply:
instead of keeping logic 'if (azureFiles && premium)` in two places (ARM and Scripts), can we define one more variable in ARM template and pass that parameter to script and directly use that?

Reply:
change the decision making to azuredeploy.json ARM template.

Thanks for the suggestion!